app/dashboard/views.py

```diff
@@ -1518,7 +1519,21 @@ def change_bounty(request, bounty_id):
         for key in keys:
             value = params.get(key, '')
             old_value = getattr(bounty, key)
+
+            if key == 'bounty_reserved_for_user' and value is not None:
+                if old_value is not None:
+                    old_value = old_value.id
+                else:
+                    old_value = None
+
+            if key == 'bounty_reserved_for_user' and value is None:
```

Comment:
Same as above.  This will never be `None`, but rather `''`
In python, `None != ''`.

If you had:


```python
val = ''
val2 = None

if not val:
    # This scope will be reached.
    print('in scope')

# Same here
if not val2:
    # In scope
    print('in scope')

# Not here though
if val is None:
    # Not in scope
    print('You never see me')
```

Also, this can be done with an if/elif vs the double if blocks.